### DISCUSSION TITLE
添加 LinkedIn 社交链接

### DISCUSSION THREAD

**Discussion by hchen98** [tangly1024/NotionNext#460] [Ideas]
有谁知道怎么可以在 `blog.config.js`  里添加 LinedIn 社交链接?

我试着添加进去后在编译的时候会有错误。

**Comment by tangly1024** [comment]
很遗憾 目前还不支持LinkIn的社交链接，不过我会将这个功能的开发列入下一版本的计划中😀

**Reply by hchen98** [reply]
唉 😔

**Comment by tangly1024** [comment]
https://github.com/tangly1024/NotionNext/issues/461
列入开发计划，将在 [3.6.0 版本](https://github.com/tangly1024/NotionNext/milestone/4)中一同上线
